src/aap_eda/services/activation/engine/podman.py

```diff
@@ -285,6 +285,13 @@ def update_logs(self, container_id: str, log_handler: LogHandler) -> None:
                 log_parts = log.split(" ", 1)
                 last_timestamp = log_parts[0]
                 if len(log_parts) > 1:
+                    message = log_parts[1]
+
+                    if "ERROR" in message:
+                        LOGGER.error(message)
+                    else:
```

Comment:
we should also check for WARN, DEBUG, FATAL, and DEBUG. It's not good to log them as INFO level.